engine/src/main/java/org/terasology/engine/module/ModuleManagerImpl.java

```diff
@@ -83,6 +85,11 @@ public ModuleManagerImpl() {
 
         setupSandbox();
         loadEnvironment(Sets.newHashSet(engineModule), true);
+        installManager = new ModuleInstallManager(this, masterServerAddress);
+    }
+
+    public ModuleManagerImpl(Config config) {
+        this(config.getNetwork().getMasterServer());
```

Comment:
Ah. That's why. I guess this should be changed, but might affect existing config JSON docs, right?